lib/instrumentation/langchain/vectorstore.js

```diff
@@ -11,16 +11,40 @@ const { LangChainVectorSearch, LangChainVectorSearchResult } = require('../../ll
 const { recordEvent, shouldSkipInstrumentation } = require('./common')
 const { DESTINATIONS } = require('../../config/attribute-filter')
 const { RecorderSpec } = require('../../shim/specs')
-const { LlmErrorMessage } = require('../../llm-events/openai')
+const LlmErrorMessage = require('../../llm-events/error-message')
+
+/**
+ * Generates a LangChainVectorSearch for entire search request.
+ * Also iterates over documents in output and generates a
+ * LangChainVectorSearchResult for each document.
+ *
+ * @param {string} request vector search query
+ * @param {number} k vector search top k
+ * @param {object} output vector search documents
+ * @param {Agent} agent NR agent instance
+ * @param {TraceSegment} segment active segment from vector search
+ * @param {string} pkgVersion langchain version
+ * @param {err} err if it exists
+ */
+// eslint-disable-next-line max-params
+function recordVectorSearch(request, k, output, agent, segment, pkgVersion, err) {
```

Comment:
I would update this to pass the params as an object. Our rule of thumb is > 2 pass in an object with relevant keys